tests/layer_formatting_tests.py

```diff
@@ -7,6 +9,15 @@
 
 class LayerFormattingTests(XMLBuilderMixin, TestCase):
     def test_build_header(self):
+        """
+        I think this header is supposed to look like this:
+
+
+        |1-1|1-2                    |
+        |   |2-1        |2-2        |
+        |   |3-1|3-2|3-3|3-4|3-5|3-6|
```

Comment:
I don't think so; Github-flavored Markdown, which is what we produce for the text output, doesn't actually support rowspan or colspan as far as I can tell. So regulations-parser includes the extra table_data field to give regulations-site hints about how to construct the HTML for more complex tables—and we can't test that from within regulations-parser.